CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### citus-docker v10.0.2.docker (March 4, 2021) ###
 
 * Bump Citus version to 10.0.2
+* Bump PostgreSQL version to 13.2
```

Comment:
we need a new line between them (e.g as we did in v9.5.0), maybe I should have wrote this more clear, sorry